### PR TITLE
feat(i18n): add `Kazakh` language

### LINKED ISSUE #3731: 🚀 - Add Kazakh language
### Which @taiga-ui/* package(s) are relevant/releated to the feature request?

i18n

### Description

Hi! I am going to add Kazakh language
based on this manual https://github.com/Tinkoff/taiga-ui/tree/main/projects/i18n

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Closes #3731

### CODE REVIEW

File: projects/i18n/interfaces/language-names.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export type TuiLanguageName =
     | 'french'
     | 'german'
     | 'italian'
+    | 'kazakhstan'
```
Comment (nsbarsukov) [inline projects/i18n/interfaces/language-names.ts:9]: `kazakh` ?

File: projects/i18n/languages/kazakh/kazakhstan.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import {TUI_KAZAKH_LANGUAGE_ADDON_COMMERCE} from './addon-commerce';
+import {TUI_KAZAKH_LANGUAGE_ADDON_EDITOR} from './addon-editor';
+import {TUI_KAZAKH_LANGUAGE_ADDON_PREVIEW} from './addon-preview';
+import {TUI_KAZAKH_LANGUAGE_ADDON_TABLE} from './addon-table';
+import {TUI_KAZAKH_LANGUAGE_CORE} from './core';
+import {TUI_KAZAKH_LANGUAGE_KIT} from './kit';
+
+export const TUI_KAZAKH_LANGUAGE: TuiLanguage = {
+    name: `kazakhstan`,
```
Comment (nsbarsukov) [inline projects/i18n/languages/kazakh/kazakhstan.ts:11]:
```suggestion
    name: `kazakh`,
```

File: projects/i18n/languages/kazakh/kazakhstan.ts
Comment (nsbarsukov) [inline projects/i18n/languages/kazakh/kazakhstan.ts:1]: `kazakh.ts`

**Review by nsbarsukov — APPROVED** [review]

File: projects/i18n/languages/kazakh/kazakhstan.ts (hunk 2 above)
Comment (splincode) [reply projects/i18n/languages/kazakh/kazakhstan.ts]: fixed

File: projects/i18n/languages/kazakh/kazakhstan.ts
Comment (splincode) [reply projects/i18n/languages/kazakh/kazakhstan.ts:1]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**